configure.ac

```diff
@@ -400,6 +400,18 @@ case "$host" in
 		use_sigposix=yes
 		with_sigaltstack=no
 		;;
+	*-*-fuchsia*)
+		AC_DEFINE(DISABLE_PORTABILITY,1,[Disable the io-portability layer])
+		AC_DEFINE(TARGET_FUCHSIA,1,[Targeting the Fuchsia platform])
```

Comment:
I think this should be `HOST_FUCHSIA`.  host is the machine mono runs on, target is the machine the aot compiler produces output for.